source/stylesheets/_header.scss

```diff
@@ -1,188 +1,213 @@
-}
-
-.header a:hover {
-  border-bottom: none;
-}
-
-// The styles bellow were ported from the compiled sass from
-// https://guides.emberjs.com/v2.3.0/
-
-// Ideally we will install bourbon / bittters / neat
-// and reuse the same _base_ stylesheets as the guides.
-// Since that will affect many default styles hence
-// the overall visual aesthetics of the site, this is an intentional
-// intermediate step to quickly port the responsive header.
-
-.header {
+header {  
   background: #dd6a58 url("../images/header.svg") top center no-repeat;
   background-size: cover;
   color: #fff;
-  padding: 1rem 0;
-}
-
-.header .container {
-  display: flex;
-  display: -webkit-flex;
-}
-
-@media screen and (max-width: 53.75rem) {
-  .responsive .header .container {
-    display: block;
-  }
-}
-
-.header .header-nav {
-  align-items: center;
-  display: flex;
-  justify-content: space-between;
-  height: 2.8rem;
-}
-@media screen and (max-width: 53.75rem) {
-  .responsive .header .header-nav {
-    height: auto;
-  }
-}
-
-.header a {
-  color: #fff;
-  font-size: 1rem;
-}
-
-@media screen and (max-width: 53.75rem) {
-  .responsive .header a {
-    line-height: 24px;
-  }
-}
-
-.header-nav li {
-  margin-top: 11px;
-  margin-bottom: 10px;
-  text-align: center;
-}
-
-@media screen and (max-width: 53.75rem) {
-  .responsive .header-nav li {
-    margin-top: 10px;
-  }
-}
-
-// Desktop
-.header-nav li {
-  -webkit-box-flex: 1;
-  -moz-box-flex: 1;
-  box-flex: 1;
-  -webkit-flex: 1 0 0;
-  -moz-flex: 1 0 0;
-  -ms-flex: 1 0 0;
-  flex: 1 0 0;
-  display: inline-block;
-}
-
-@media screen and (max-width: 53.75em) {
-  .responsive .header-nav li {
-    display: inherit;
-  }
-}
-@media screen and (max-width: 53.75em) {
-  .responsive .header-nav li:not(.header-logo):not(.header-search) {
-    width: 32%;
-    display: inline-block;
-  }
-}
-
-
-.header-nav li:not(.header-logo):not(.header-search) {
-  width: auto;
-}
-@media screen and (min-width: 53.75rem) {
-}
-
-.header-nav .header-logo {
-  -webkit-box-flex: 1.5;
-  -moz-box-flex: 1.5;
-  box-flex: 1.5;
-  -webkit-flex: 1.5 0 0;
-  -moz-flex: 1.5 0 0;
-  -ms-flex: 1.5 0 0;
-  flex: 1.5 0 0;
-}
-@media screen and (min-width: 53.75rem) {
-}
-
-.header-nav .header-search {
-  -webkit-box-flex: 2;
-  -moz-box-flex: 2;
-  box-flex: 2;
-  -webkit-flex: 2 0 0;
-  -moz-flex: 2 0 0;
-  -ms-flex: 2 0 0;
-  flex: 2 0 0;
-}
-
-@media screen and (max-width: 53.75rem) {
-  .responsive .header-nav .header-search {
-    display: block;
-  }
-}
-
-.header-logo a {
-  height: 40px;
-  width: 100px;
-  background: url("../images/ember-logo.svg") no-repeat;
-  display: block;
-  margin: -15px auto 0 0;
-}
-
-@media screen and (max-width: 53.75rem) {
-  .responsive .header-logo a {
-    margin-left: auto;
-  }
-}
-
-
-.header-search {
-  margin-left: 4%;
-
-  .algolia-autocomplete {
-    display: block !important; // beats `inline-block` added by DocSearch script
-  }
-  .search-input {
+  padding: 0;
+  nav {
```

Comment:
Styling the `nav` tag directly seems undesirable, can we semantically have only one `nav` element in our page?